src/main/java/github/grace000/erversay/RequestReader.java

```diff
@@ -0,0 +1,21 @@
+package github.grace000.erversay;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+
+public class RequestReader {
+    public String readRequest(BufferedReader bufferedReader) throws IOException {
+        String input;
+        StringBuilder requestLine = new StringBuilder();
+
+        while(true) {
```

Comment:
Is there a way to do this so the while loop is based on the readline value? So there's not a hard break